### PR TITLE
add symfony/options-resolver ^6.0

### PR DESCRIPTION
**Pull request by Jimbolino**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
         "psr/http-message": "^1.0",
         "psr/http-client": "^1.0",
         "php-http/httplug": "^1.1 || ^2.0",
-        "symfony/options-resolver": "^3.4 || ^4.0 || ^5.0",
+        "symfony/options-resolver": "^3.4 || ^4.0 || ^5.0" || ^6.0",
```
Comment (ZebulanStanphill) [inline composer.json:25]: There's a breaking typo here. Here's the fix.
```suggestion
        "symfony/options-resolver": "^3.4 || ^4.0 || ^5.0 || ^6.0",
```

**Review by ZebulanStanphill — CHANGES_REQUESTED** [review]

**Comment by ZebulanStanphill** [comment]
In case anyone is worried about breaking changes: the only listed change for version 6.0 is the [removal of `OptionsResolverIntrospector::getDeprecationMessage()`](https://github.com/symfony/options-resolver/blob/6.0/CHANGELOG.md#60), which is [not used by this project](https://github.com/kriswallsmith/Buzz/search?q=getDeprecationMessage), so `composer.json` is indeed the only thing that needs to be changed.

**Comment by Nyholm** [comment]
Thank you

**Comment by Nyholm** [comment]
https://github.com/kriswallsmith/Buzz/releases/tag/1.2.1

**Pull request merged**